tests/tests_integration_pytest.yml

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
+
+- import_playbook: playbooks/rhel7_deps.yml
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == '7'
+
+- import_playbook: playbooks/centos6_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '6'
+
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '7'
+
+- import_playbook: playbooks/centos8_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '8'
+
+- name: Run test playbook
+  hosts: all
+  vars:
+    - rundir: /run/system-roles-test
+  tasks:
+    - file:
+        state: directory
+        path: /run
+
+    - shell: |
+        # set -x
+        curdir="${PWD}"
+        gitdir=""
+        for i in {1..255}
+        do
+            candidate="${curdir}/.git"
+            if [ -d "${candidate}" ]
+            then
+                gitdir=${candidate}
+                break
+            else
+                curdir=$(dirname "${curdir}")
+            fi
+        done
+        if [ -z "${gitdir}" ]
+        then
+            exit 1
+        fi
+        echo "$(dirname ${gitdir})"
```

Comment:
IIUC what this is supposed to be doing, you can use `command: git rev-parse --show-toplevel` instead of this `shell:` script

Reply:
Awesome, good to know! Thanks!